amundsen_application/api/utils/metadata_utils.py

```diff
@@ -39,3 +26,74 @@ def marshall_table_partial(table: Dict) -> Dict:
         'type': 'table',
         'last_updated_epoch': table.get('last_updated_epoch', None),
     }
+
+
+def marshall_table_full(table: Dict) -> Dict:
+    """
+    Forms the full version of a table Dict, with additional and sanitized fields
+    :param table: Table Dict from metadata service
+    :return: Table Dict with sanitized fields
+    """
+    # Filter and parse the response dictionary from the metadata service
+    fields = [
+        'columns',
+        'cluster',
+        'database',
+        'is_view',
+        'key',
+        'owners',
+        'schema',
+        'source',
+        'table_description',
+        'table_name',
+        'table_readers',
+        'table_writer',
+        'tags',
+        'watermarks',
+        # 'last_updated_timestamp' Exists on the response from metadata but is not used.
+        # This should also be consolidated with 'last_updated_epoch' to have the same name and format.
```

Comment:
Should clean up `last_updated_timestamp` and `last_updated_epoch`.
Most notably, this value is missing from `popular_tables` schema.